@vates/fatfs/structs.js

```diff
@@ -7,7 +7,7 @@ var _ = require('struct-fu'),
 var bootBase = _.struct([
     _.byte('jmpBoot', 3),
     _.char('OEMName', 8),
-    _.uint16le('BytsPerSec'),
+    _.uint16le('BytesPerSec'),
     _.uint8('SecPerClus'),
     _.uint16le('ResvdSecCnt'),      // Rsvd in table, but Resvd in calcs…
```

Comment:
Certainly, this hints that the code is willing to rename things. I could add a comment above saying `//  BytsPerSec in something, but aliased for readability` or something...